playground/webpack.config.js

```diff
@@ -134,25 +134,23 @@ module.exports = (env = {production: false}) => ({
         test: /\.tsx?$/,
         use: [
           {
-            loader: 'awesome-typescript-loader',
+            loader: 'babel-loader',
```

Comment:
Is this because we were applying babel through the ts loader before, so now we need to do this manually?

Reply:
As a follow-up, have you looked into whether we can just use babel to compile TS?

Reply:
Bingo. I havent looked into using Babel for compiling TS, as I figured sticking close to what SK does under the hood shall result in the least confusion.

Reply:
I saw TS recently announced that they fully support babel, would be cool if we could use that to simplify build configs